other-docs/guides/upgrading/v25.md

```diff
@@ -0,0 +1,86 @@
+---
+order: 25
+---
+
+# Upgrading to v25
+
+*If you are migrating from WordPress to Altis, check out the [migrating guide](../migrating/) first.*
+
+To upgrade to Altis v25, edit your `composer.json` and change the version constraint for `altis/altis` and any local environment
+modules to `^25.0.0`.
+
+```json
+{
+    "require": {
+        "altis/altis": "^25.0.0"
+    },
+    "require-dev": {
+        "altis/local-server": "^25.0.0"
+    },
+    "config": {
+        "platform": {
+            "php": "8.3"
+        }
+    }
+}
+```
+
+Once you have made these changes, run `composer update` and then run the `wp altis migrate` command:
+
+```sh
+# For cloud environments
+wp altis migrate
+
+# For local server
+composer server cli -- altis migrate
+```
+
+## Headline Features
+
+### Support for PHP 8.4
+
+Altis v25 introduces support for PHP 8.4. It will first be available for testing in local server, then made available for upgrade
+on development, staging and production environments.
+
+#### Breaking changes:
+
+- Deprecation of `php-v8js`: with the release of Altis v25, `php-v8js` is removed for PHP 8.4 and deprecated for earlier PHP
+versions.
+- All required dependencies have been updated to support the new version of PHP.
+- You might still see a couple of warnings here and there, but we are working on fixing them.
+- We have updated the default PHP version to 8.3.
+
+### Altis Dashboard Features
+
+#### Self-managed Allow list / Block list
+
+You can now add individual IP addresses and ranges to allow lists to make sure they are never filtered.
```

Comment:
Is "filtered" the right word? I've only heard talk of rate limiting. If we only do rate limiting we should probably say "make sure they are not rate limited". But if we do other checks, filtered is fine.
Either way, I. think it should be "not" instead of "never".